src/pages/workspaces/workspace/Dashboard/WorkspaceTags.test.ts

```diff
@@ -0,0 +1,183 @@
+import { DeepPartial } from '@terra-ui-packages/core-utils';
+import { act, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import { Ajax } from 'src/libs/ajax';
+import { WorkspaceTags } from 'src/pages/workspaces/workspace/Dashboard/WorkspaceTags';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+import { defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+
+type AjaxContract = ReturnType<typeof Ajax>;
+type AjaxExports = typeof import('src/libs/ajax');
+
+jest.mock('src/libs/ajax', (): AjaxExports => {
+  return {
+    ...jest.requireActual('src/libs/ajax'),
+    Ajax: jest.fn(),
+  };
+});
+
+// set the collapsable panel to be open
+jest.mock('src/libs/prefs', (): typeof import('src/libs/prefs') => ({
+  ...jest.requireActual('src/libs/prefs'),
+  getLocalPref: jest.fn().mockReturnValue(true),
+}));
+
+type ErrorExports = typeof import('src/libs/error');
+jest.mock(
+  'src/libs/error',
+  (): ErrorExports => ({
+    ...jest.requireActual('src/libs/error'),
+    reportError: jest.fn(),
+  })
+);
+
+jest.mock('react-notifications-component', () => {
```

Comment:
I don't think you actually need to mock this (I tried commenting it out, and the test passed).

The need was likely just transitory when the code/test was in a state where errors were being thrown.